node/Cargo.toml

```diff
@@ -40,10 +40,8 @@ assert_matches = "1.5.0"
 async-trait = "0.1.71"
 bit-vec = "0.6"
 blst = "0.3.10"
-ark-bn254 = "0.4.0"
-ark-ec = "0.4.2"
-ark-serialize = { version = "0.4.2", features = ["std"] }
-num-traits = "0.2.17"
+pairing = { package = "pairing_ce", git = "https://github.com/moshababo/pairing.git" }
```

Comment:
wouldn't it be easier to push a temporary branch to matterlabs pairing repo? Also please pin to a commit

Reply:
It was a temp solution, already removed. 

Commit is pinned in `Cargo.lock`, is it enough?

Reply:
No, cargo.lock will get updated whenever you compile the crate. If you want to pin to a particular commit, you need to do it on cargo.toml.

Reply:
I added the git rev.